package.json

```diff
@@ -0,0 +1,44 @@
+{
+  "name": "fluent-schema",
+  "version": "0.1.0",
```

Comment:
we typically bump the version number when doing the release, this should be something like `0.0.1`, and we'll bump `0.1.0`  when releasing.